geoclue/meson.build

```
@@ -1,5 +1,8 @@
-install_data(
-    'beacondb.conf',
-    rename: '99-beacondb.conf',
-    install_dir: sysconfdir / 'geoclue' / 'conf.d'
-)
+# Configure GeoClue to use BeaconDB
+if get_option('appcenter-flatpak')
```

```suggestion
if get_option('geoclue')
```

Looks like a copy & paste error here :sweat_smile:

---

Woops yup ha